config/routes.rb

```diff
@@ -11,5 +11,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :public_recipes
```

Comment:
- [ ]  You can put this with all the other routes, this is just aesthetic for better organization [Optional]